BUILDING.md

```diff
@@ -468,3 +468,29 @@ export SWIFTSHADER_LD_LIBRARY_PATH=`pwd`
 ```
 
 Next, go to your Filament repo and use the [easy build](#easy-build) script with `-t`.
+
+## SwiftShader for CI
+
+Continuous testing turnaround can be quite slow if you need build SwiftShader from scratch, so we
+provide an Ubuntu-based Docker image that has it ready to go. The Docker image also includes
+everything necessary for building Filament and running tests. You can fetch and run the image as
+follows:
+
+```
+docker pull ghcr.io/prideout/ssfilament
```

Comment:
Should we create a `filament` account?

Reply:
Someone already has that namespace.  It may make sense to use the "google" account but I have not figured out how to get permission for that.

Reply:
Created an account called `filament-assets` for this.